jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -36,7 +37,7 @@ def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmpdir):
     assert len(mv_data) == 1
     assert mv_data[0].label == 'moment 0'
 
-    assert len(dc.links) == 8
+    assert len(dc.links) == 14
```

Comment:
Do you know why there are so many more links? Is it because the moment map now has WCS and WCS uses more links?

Reply:
That is my guess. Before this, moment map has no WCS, and we hacked Cubeviz to secretly grab the WCS from the "flux" cube.

Reply:
But why 14 and not 10, that I don't know. Someone who knows how to read the links mini-language can tell me.

Reply:
If you run the test case in an interactive notebook, you can get out the following. Does this look about right? Or is something off?

```python
for i, link in enumerate(dc.links):
    print(f"{i+1}: " + link.to_html() + "<br/>")
```

<div>
1: <font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x] &#8592; WCSLink.forwards_1(<font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y], <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z])<br/>
2: <font color='#777777'>[test[FLUX]]</font>.Declination &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z], <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y])<br/>
3: <font color='#777777'>[test[FLUX]]</font>.Wave &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Pixel Axis 2 [x])<br/>
4: <font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y] &#8592; using(<font color='#777777'>[moment 0]</font>.Declination, <font color='#777777'>[moment 0]</font>.Right Ascension)<br/>
5: <font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x] &#8592; using(<font color='#777777'>[moment 0]</font>.Declination, <font color='#777777'>[moment 0]</font>.Right Ascension)<br/>
6: <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z] &#8592; WCSLink.backwards_2(<font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x], <font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y])<br/>
7: <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z] &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Right Ascension, <font color='#777777'>[test[FLUX]]</font>.Declination)<br/>
8: <font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y] &#8592; WCSLink.forwards_2(<font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y], <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z])<br/>
9: <font color='#777777'>[moment 0]</font>.Declination &#8592; using(<font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y], <font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x])<br/>
10: <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y] &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Right Ascension, <font color='#777777'>[test[FLUX]]</font>.Declination)<br/>
11: <font color='#777777'>[moment 0]</font>.Right Ascension &#8592; using(<font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y], <font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x])<br/>
12: <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 2 [x] &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Wave)<br/>
13: <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y] &#8592; WCSLink.backwards_1(<font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x], <font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y])<br/>
14: <font color='#777777'>[test[FLUX]]</font>.Right Ascension &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z], <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y])<br/>
</div>

Reply:
The original 8:

<div>
2: <font color='#777777'>[test[FLUX]]</font>.Declination &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z], <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y])<br/>
3: <font color='#777777'>[test[FLUX]]</font>.Wave &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Pixel Axis 2 [x])<br/>
6: <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z] &#8592; WCSLink.backwards_2(<font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x], <font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y])<br/>
7: <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z] &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Right Ascension, <font color='#777777'>[test[FLUX]]</font>.Declination)<br/>
10: <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y] &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Right Ascension, <font color='#777777'>[test[FLUX]]</font>.Declination)<br/>
12: <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 2 [x] &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Wave)<br/>
13: <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y] &#8592; WCSLink.backwards_1(<font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x], <font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y])<br/>
14: <font color='#777777'>[test[FLUX]]</font>.Right Ascension &#8592; using(<font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z], <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y])<br/>
</div

Additional 6:

<div>
1: <font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x] &#8592; WCSLink.forwards_1(<font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y], <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z])<br/>
4: <font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y] &#8592; using(<font color='#777777'>[moment 0]</font>.Declination, <font color='#777777'>[moment 0]</font>.Right Ascension)<br/>
5: <font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x] &#8592; using(<font color='#777777'>[moment 0]</font>.Declination, <font color='#777777'>[moment 0]</font>.Right Ascension)<br/>
8: <font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y] &#8592; WCSLink.forwards_2(<font color='#777777'>[test[FLUX]]</font>.Pixel Axis 1 [y], <font color='#777777'>[test[FLUX]]</font>.Pixel Axis 0 [z])<br/>
9: <font color='#777777'>[moment 0]</font>.Declination &#8592; using(<font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y], <font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x])<br/>
11: <font color='#777777'>[moment 0]</font>.Right Ascension &#8592; using(<font color='#777777'>[moment 0]</font>.Pixel Axis 0 [y], <font color='#777777'>[moment 0]</font>.Pixel Axis 1 [x])<br/>
</div>

Reply:
It looks like there are two links between the moment and flux data pixel axes, two mapping pixel to RA/Dec, and two mapping RA/Dec to pixel. Considering similar pixels <-> RA/Dec links exist for the flux data, this doesn't worry me (despite not 100% understanding all of the underlying linking logic). I'd be ok merging.